incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -284,6 +283,30 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(hasObject(ctx, "CronTab", barName, "foo-crontab")).Should(BeTrue())
 		Expect(objectInheritedFrom(ctx, "CronTab", barName, "foo-crontab")).Should(Equal(fooName))
 	})
+
+	It("should set NumPropagatedObjects back to 0 after deleting the source object in propagate mode", func() {
+		addToHNCConfig(ctx, "v1", "LimitRange", api.Propagate)
+		setParent(ctx, barName, fooName)
+		makeObject(ctx, "LimitRange", fooName, "foo-lr")
+
+		Eventually(getNumPropagatedObjects(ctx, "v1", "LimitRange"), 4*time.Second).Should(Equal(int32(1)))
```

Comment:
Replace `4*time.Second` with a named constant with an explanation of how you picked that number

Reply:
Good catch! Done!